statistik.html

```diff
@@ -55,15 +55,40 @@
     <main>
       <section class="statistic">
         <div class="container">
+          <nav class="nav-side"><a href="index.html">Start</a> <i class="fas fa-chevron-right"></i> Statistic</nav>
            <h2>Statistik</h2>     
             <img src="statistik_webb.png" alt="map with human" class="float-img"/>
```

Comment:
you should put your image in the statistics-folder 
![image](https://user-images.githubusercontent.com/72127499/97690565-6d3f8880-1a9d-11eb-9227-963338ea8e52.png)